base/sysimg.jl

```diff
@@ -1,6 +1,12 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-Core.include(Main, "Base.jl")
+# Can be built either a monolith or with a minimal Base image that just has the
+# compiler.
```

Comment:
Do we actually need this extra bootstrapping complexity for any reason? No reason not too I guess, but seems quite tedious for future people to understand what is going on when they try to make changes